src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.DataConverter;
+import service.FileReader;
+import service.FileWriter;
+import service.ReportGenerator;
+import service.ShopService;
+import service.impl.DataConverterImpl;
+import service.impl.FileReaderImpl;
+import service.impl.FileWriterImpl;
+import service.impl.ReportGeneratorImpl;
+import service.impl.ShopServiceImpl;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+import strategy.impl.BalanceOperation;
+import strategy.impl.OperationStrategyImpl;
+import strategy.impl.PurchaseOperation;
+import strategy.impl.ReturnOperation;
+import strategy.impl.SupplyOperation;
+
+public class Main {
+    private static final String INPUT_FILE = "src/main/resources/reportToRead.csv";
```

Comment:
Avoid using hardcoded file paths. It's better to pass them as parameters to the main method or to read them from a configuration file or environment variable.

Reply:
I agree, but in the task we shouldn't do this.

Reply:
At the level of Main class it's ok